simpleai_base/models_info.py

```
@@ -676,13 +677,20 @@ def get_model_filepath(self, catalog='', model_path='', muid=None):
                 return self.m_info[model_key]['file'][0]
         return ''
 
-    def get_model_names(self, catalog, filters=[], casesensitive=False, reverse=False):
+    def get_model_names(self, catalog, filters=[], casesensitive=False, reverse=False, user=None):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)